tests/test_file.py

```diff
@@ -74,11 +77,23 @@ def _create_test_credentials(self, client_id='some_client_id',
             user_agent)
         return credentials
 
-    def test_non_existent_file_storage(self):
+    @mock.patch('warnings.warn')
+    def test_non_existent_file_storage(self, warn_mock):
         s = file.Storage(FILENAME)
```

Comment:
That syntax is used throughout this file.  Should I update them all in this PR, even though they aren't related to this change?